### PR TITLE
Bump libsigc++ to 2.10.0

### PR DESCRIPTION
**Pull request by alarcher**
ABI compatible: https://abi-laboratory.pro/tracker/timeline/libsigc++/

### CODE REVIEW

File: components/library/libsigc++/test/results-32.master [hunk 1]
```diff
@@ -0,0 +1,39 @@
+PASS: test_accum_iter
```
Comment (pyhalov) [inline components/library/libsigc++/test/results-32.master]: Why should we ship 2 files if results match?

Comment (alarcher) [reply components/library/libsigc++/test/results-32.master]: I can merge them if you want.

Comment (pyhalov) [reply components/library/libsigc++/test/results-32.master]: it would be better

Comment (pyhalov) [reply components/library/libsigc++/test/results-32.master]: Otherwise, LGTM

Comment (alarcher) [reply components/library/libsigc++/test/results-32.master]: Fixed.

**Pull request merged**